contracts/StakingPoolV2.sol

```diff
@@ -231,6 +235,10 @@ contract StakingPoolV2 is IStakingPoolV2, StakedElyfiToken, Ownable {
     SafeERC20.safeTransfer(rewardAsset, _msgSender(), residueAmount);
   }
 
+  function setManager(address managerAddr) external onlyOwner {
+    _setupRole(MANAGER_ROLE, managerAddr);
+  }
```

Comment:
If you set the DEFAULT_ROLE_ADMIN in the constructor by `_setupRole(DEFAULT_ADMIN_ROLE, msg.sender);`, you can just use `grantRole` of the `AccessControl` contract without this function.

Reply:
👍